### ISSUE TITLE
wandb 0.8.16 is incompatible with graphql-core==0.3.0

### ISSUE THREAD

**Issue by ayushkovskiy** [wandb/client#692]
Error: version conflict, wandb cannot start with `graphql-core==0.3.0`:
```
wandb --version
Traceback (most recent call last):
  File "/usr/local/bin/wandb", line 5, in <module>
    from wandb.cli import cli
  File "/usr/local/lib/python3.6/dist-packages/wandb/__init__.py", line 45, in <module>
    from wandb.apis import InternalApi, PublicApi, CommError
  File "/usr/local/lib/python3.6/dist-packages/wandb/apis/__init__.py", line 7, in <module>
    from gql.client import RetryError
  File "/usr/local/lib/python3.6/dist-packages/gql/__init__.py", line 2, in <module>
    from .client import Client
  File "/usr/local/lib/python3.6/dist-packages/gql/client.py", line 3, in <module>
    from graphql import parse, introspection_query, build_ast_schema, build_client_schema
ImportError: cannot import name 'introspection_query'
```

`wandb --version && python --version && uname`

* Weights and Biases version:
```
# pip show wandb
Name: wandb
Version: 0.8.16
Summary: A CLI and library for interacting with the Weights and Biases API.
Home-page: https://github.com/wandb/client
Author: Weights & Biases
Author-email: support@wandb.com
License: MIT license
Location: /usr/local/lib/python3.6/dist-packages
Requires: shortuuid, nvidia-ml-py3, watchdog, GitPython, sentry-sdk, docker-pycreds, six, subprocess32, python-dateutil, gql, Click, configparser, requests, psutil
Required-by: 
```

* Python version:
```
# python --version 
Python 3.6.8
```

* Operating System:
```
# uname -a
Linux job-4ab92e1c-250c-42bf-9d64-2f610d7a6c64 4.14.137+ #1 SMP Thu Aug 8 02:47:02 PDT 2019 x86_64 x86_64 x86_64 GNU/Linux
```
### Description

Dependencies:
`wandb` requires `gql` requires `graphql-core`:
```
# pip show wandb
Name: wandb
Version: 0.8.16
Summary: A CLI and library for interacting with the Weights and Biases API.
Home-page: https://github.com/wandb/client
Author: Weights & Biases
Author-email: support@wandb.com
License: MIT license
Location: /usr/local/lib/python3.6/dist-packages
Requires: six, watchdog, configparser, shortuuid, docker-pycreds, requests, nvidia-ml-py3, python-dateutil, subprocess32, sentry-sdk, GitPython, psutil, Click, gql
Required-by: 

# pip show gql
Name: gql
Version: 0.1.0
Summary: GraphQL client for Python
Home-page: https://github.com/graphql-python/gql
Author: Syrus Akbary
Author-email: me@syrusakbary.com
License: MIT
Location: /usr/local/lib/python3.6/dist-packages
Requires: six, promise, graphql-core
Required-by: wandb

# pip show graphql-core
Name: graphql-core
Version: 3.0.0
Summary: GraphQL implementation for Python, a port of GraphQL.js, the JavaScript reference implementation for GraphQL.
Home-page: https://github.com/graphql-python/graphql-core-next
Author: Christoph Zwerschke
Author-email: cito@online.de
License: MIT license
Location: /usr/local/lib/python3.6/dist-packages
Requires: 
Required-by: gql
```

Note, `wandb==0.8.16` works well together with `gql==0.1.0` and `graphql-core==2.2.1`.

## Solution:
If `wandb` really needs `graphql-core<0.3.0`, I propose to add this requirement explicitly to wandb's `setup.py` in addition to [exitsting dependency on `gql`](https://github.com/wandb/client/blob/7bbe47232c30067643157525969edf88013ad1c9/setup.py#L12) as `gql` itself [does not have this restriction](https://github.com/graphql-python/gql/blob/3653bb5260b60a6c72d0bb0137874fb40969a826/setup.py#L29).

**Comment by vanpelt** [comment]
Thanks for reporting!  We saw this as well over the weekend.  We have a new release that should go out today or tomorrow, you can install with: `pip install --upgrade git+git://github.com/wandb/client.git@task/pin_graphql#egg=wandb`

**Comment by vanpelt** [comment]
We just pushed a fix for this in 0.8.17.
